tests/integration/tier1/yum-distro-sync/test_yum_distro_sync.py

```diff
@@ -0,0 +1,66 @@
+from envparse import env
+
+
+def test_yum_distro_sync(convert2rhel, shell):
+    """Test yum distro-sync command edge-cases when given packages aren't in enabled repositories.
+
+    Following is done:
+        1) Install a problematic package (cpaste) - done in preparation step
+        2) Run the conversion
+        3) Run the distro-sync in two variants
+            - just with the problematic package
+            - with some another package, which is in RHEL repositories
+        4) Do the same checks and conditions as is in convert2rhel/pkghandler.py in call_yum_cmd_w_downgrades.
+
+    Another problem is, that yum behaves differently on Centos 7 and Centos 8
+        - on Centos 7 gives 0 and any error in both cases
+        - on Centos 8  gives 0 and any error if in list of packages for distro-sync is at least
+          one, which can be successfully distro synced. If all of the given cannot be synced, there
+          is an error, which caused problems: https://issues.redhat.com/browse/OAMG-4600. But
+          the error isn't in fact error, the package stays there
+          and just isn't supported by Red Hat.
+    """
+
+    with convert2rhel(
+        ("-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_USERNAME"),
+            env.str("RHSM_PASSWORD"),
+            env.str("RHSM_POOL"),
+        ),
+        unregister=False,
+    ) as c2r:
+        c2r.expect("Conversion successful!")
+    assert c2r.exitstatus == 0
+
+    out = shell("yum distro-sync cpaste zip")  # any error on Centos 7 and Centos 8
```

Comment:
I would move the comment above so it's consistent in the code.
```suggestion
    # any error on Centos 7 and Centos 8
    out = shell("yum distro-sync cpaste zip")  
```